web/themes/custom/pineapple/templates/paragraphs/paragraph--components.html.twig

```diff
@@ -48,11 +48,13 @@
 %}
 {% block paragraph %}
   <div{{ attributes.addClass(classes) }}>
-    {% include "@components/_heading.twig" with {
-      heading_level: 2,
-      heading_text: content.field_p_c_heading|field_value|first['#context'].value,
-      heading_attributes: create_attribute().addClass(paragraph.field_p_c_visually_hide_heading.value == 1 ? 'visually-hidden')
-    } %}
+    {% if content.field_p_c_heading|field_value %}
```

Comment:
Checks like this should be done like "not entity.field_name.isEmpty()"

Reply:
Done.